index.html

```diff
@@ -167,7 +167,7 @@ <h2 class="flush-top font-size-base text-gray-light">
         {% if managed_features.size > 0 %}
         featured
         {% else %}
-        latest articles
+          latest
```

Comment:
This is the change to display "latest" but it is only triggered when there are no managed features

Reply:
Makes sense! Thanks for letting me know.